flask_security/core.py

```diff
@@ -136,6 +136,7 @@
     ],
     'DEPRECATED_HASHING_SCHEMES': ['hex_md5'],
     'DATETIME_FACTORY': datetime.utcnow,
+    'USER_ACTIVE_BY_DEFAULT': True,
```

Comment:
Can you please move it close to `USER_IDENTITY_ATTRIBUTES` option?

Reply:
Doesn't `USER_IDENTITY_ATTRIBUTES` specify what user attributes uniquely identify it?

If so, I didn't meant this thing to be a user-specific attribute but rather an application-wise configuration that determines whether or not new users are active by default therefore being able to login without being reviewed.

Thing of this way: you can set `USER_ACTIVE_BY_DEFAULT` to True and you'll have a standard registration, as soon as you register and confirm your email you can login.
On the other hand, if you set `USER_ACTIVE_BY_DEFAULT` to False, after you register you will have to wait for administrators to approve your account  before  being able to log in.

It's basically designed for restricted-access websites.

Reply:
It sounds good. I was a bit confused by the name. Can you add the documentation and consider changing the name to something around:

- `MANUAL_USER_ACTIVATION`
- `RESTRICTED_REGISTRATION`
...

Reply:
Sure! Checkout the new commit